src/components/Auth/TodosHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { memo, useEffect, useRef } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  newTodoTitle: string;
+  setNewTodoTitle: (title: string) => void;
+  onAddNewTodo: (event: React.FormEvent) => void;
+  isAdded: boolean;
+  onChangeAllTodos: () => void;
+  isAllTodosCompleted: boolean;
+};
+
+export const TodosHeader: React.FC<Props> = memo(({
+  newTodoTitle,
+  setNewTodoTitle,
+  onAddNewTodo,
+  isAdded,
+  onChangeAllTodos,
+  isAllTodosCompleted,
+}) => {
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
```

Comment:
The input should always have a focus after adding
```suggestion
  }, [isAdded]);
```

Reply:
done